src/data/real/ereal.lean

```diff
@@ -62,6 +62,9 @@ def real.to_ereal : ℝ → ereal := some ∘ some
 
 namespace ereal
 
+-- Provide explicitly, otherwise it is inferred noncomputably from `complete_linear_order`
+instance : has_bot ereal := ⟨some ⊥⟩
```

Comment:
Are you sure you aren't causing a diamond doing this? `ereal.complete_linear_order` has already been defined by that point so its bot conflicts with the one you're providing here. The solution here is to go through `with_top.order_bot` and not automatically derive `complete_linear_order`.

Reply:
Can you elaborate? This is precisely the `bot` provided by `complete_linear_order`. However, that whole class is tainted by noncomputability, so even this computable data is tagged so.

Reply:
Ah okay, I assumed it meant `⊥` was defined using some Sup/Inf dirtying. I don't know how computability behaves in those case but I assume Lean looks for computable instances first?